presto-docs/src/main/sphinx/release/release-0.220.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+* Fix an issue when executing spatial join on top of a right join. See :issue:`12672`.
```

Comment:
Clarify what kind of issue (that it's a query failure, rather than a correctness or performance issue). e.g.
```
Fix an issue where spatial joins executed after right outer joins would fail in certain cases.
```